providers/linkedin_test.go

```diff
@@ -98,8 +99,9 @@ func TestLinkedInProviderGetEmailAddress(t *testing.T) {
 	bURL, _ := url.Parse(b.URL)
 	p := testLinkedInProvider(bURL.Host)
 
+	ctx := context.TODO()
```

Comment:
Please do the others in this file too
```suggestion
	ctx := context.Background()
```